src/encoded/loadxl.py

```diff
@@ -465,13 +465,11 @@ def load_data(app, indir='inserts', docsdir=None, overwrite=False,
         indir (inserts): inserts folder, should be relative to tests/data/
         docsdir (None): folder with attachment documents, relative to tests/data
     '''
-    from webtest import TestApp
     environ = {
         'HTTP_ACCEPT': 'application/json',
         'REMOTE_USER': 'TEST',
     }
-    testapp = TestApp(app, environ)
-    from pkg_resources import resource_filename
+    testapp = webtest.TestApp(app, environ)
```

Comment:
No reason not to use `VirtualApp` at this point, no?

Reply:
Yeah, I noticed this on the way through and thought the same thing but I was just doing largely syntactic changes and didn't want to look up the details of how to use VirtualApp at that moment. I agree in general and can probably circle back to do that now that I'm out of the pit.

Reply:
I've got a change to this uploading in the next commit.

Reply:
Hmm. That tested poorly. Either some intermittents that were unrelated or something odd about that class. I'm gonna back out of that change and do it another time when there's less going on.